### PR TITLE
Seed issues

### PR DESCRIPTION
**Pull request by Cajam**
Did fine up to right after the Active Record portion. Can't get my seeds.rb file to inject, and so I can't get my code to work properly, even after I felt good moving forward after Tunr.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  erb :"index"
+end
+
+# get the apartments href. Took me a couple hours to figure out you have to wrap "apartments/index" in quotes to get it to work. Funny because erb :index works without quotes, but whatever.
+get '/apartments' do
+  erb :"apartments/index"
+end
+
+get '/apartments/:id' do
+  erb :"apartments/1"
```
Comment (jshawl) [inline app.rb:22]: before the erb line:

``` rb
@apartment = Apartment.find(params[:id])
erb :"apartments/index"
```

File: db/seeds.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+blake.age = 24
+blake.save!
+
+# Update attributes with .update
+blake.update(age: 25)
+
+# Time to retire, Donovan. Philly fans will never appreciate you.
+# Not sure if I need this top line but I'll include it.
+donovan = Tenant.find_by(name: "Donovan McNabb")
+donovan.destroy
```
Comment (jshawl) [inline db/seeds.rb:44]: what happens when you run `$ ruby db/seeds.rb`?

File: models/apartment.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+class Apartment < ActiveRecord::Base #inherits from Active Record
+  has_many :tenants
+  attr_accessor :renters
+  attr_reader :address, :monthly_rent, :sqft, :numb_beds, :num_baths
```
Comment (jshawl) [inline models/apartment.rb:4]: remove the `attr` helper methods here and let Active Record define getters and setters

File: models/apartment.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  # I think I'm OK with commenting this initialize method out since I have ActiveRecord going?
```
Comment (jshawl) [inline models/apartment.rb:6]: :+1:

File: models/apartment.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  # end
+
+  # This follwing def breaks Sinatra for some reason.
+
+  # def add_tenant
+  #   if renters > 0
+  #     puts "There is already someone in this room."
+  #   else
+  #     Tenant.new  # create new person
+  #     puts "Tenant added."
```
Comment (jshawl) [inline models/apartment.rb:24]: you're missing one `end`

File: models/tenant.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+class Tenant < ActiveRecord::Base
+  belongs_to :apartment
+  attr_accessor :name, :age, :gender
+
+  def initialize(tenant_name, tenant_age, tenant_gender)
```
Comment (jshawl) [inline models/tenant.rb:5]: remove this initialize method and let Active Record handle object instantiation

**Comment by jshawl** [comment]
:+1: You're off to a great start! Let me know if you'd like to work through any of this together - I'm on office hours today

File: db/seeds.rb (hunk 2 above)
Comment (Cajam) [inline db/seeds.rb:44]: When I tried to run seeds.rb I'd get an error saying there was a misplaced
"end" in the code, correcting that would result in 11 new failures inside
the ActiveRecord source I think

Very respectfully,

_Cassidy J. Harpster_
cassidy.j.harpster@gmail.com
(850) 218-4673
Washington, D.C.

**Pull request closed without merging**